x/nexus/keeper/rate_limit.go

```diff
@@ -0,0 +1,135 @@
+package keeper
+
+import (
+	"fmt"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/axelarnetwork/axelar-core/utils"
+	"github.com/axelarnetwork/axelar-core/utils/events"
+	"github.com/axelarnetwork/axelar-core/utils/key"
+	"github.com/axelarnetwork/axelar-core/x/nexus/exported"
+	"github.com/axelarnetwork/axelar-core/x/nexus/types"
+	"github.com/axelarnetwork/utils/funcs"
+)
+
+func getRateLimitKey(chain exported.ChainName, asset string) key.Key {
```

Comment:
Public functions should be put at the top of the file and private ones at the bottom.